components/Comment/CommentEmptyState.tsx

```diff
@@ -0,0 +1,46 @@
+import colors from "./lib/colors";
+import { css, StyleSheet } from "aphrodite";
+import { NullableString, RhDocumentType } from "~/config/types/root_types";
+
+type Args = {
+  forSection: NullableString;
+  documentType: RhDocumentType;
+  height: string;
+};
+
+const CommentEmptyState = ({ forSection, documentType, height }: Args) => {
+  return (
+    <div className={css(styles.wrapper)} style={{ height }}>
+      {forSection === "REVIEW" ? (
```

Comment:
Anyway we can split this a bit better? I always find this style of switching between `forSection` a bit hard to read / maintain.

Maybe it's easier to read with a switch statement? Or instead have the parent component pass in the render state